ruleset/rules/0435-ms_logs_rules.xml

```diff
@@ -1,10 +1,8 @@
 <!--
-  -  Windows logs rules
-  -  Created by Wazuh, Inc.
-  -  Copyright (C) 2015-2021, Wazuh Inc.
-  -  This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2.
+    Copyright (C) 2015-2021, Wazuh Inc.
 -->
 
+
 <!-- ID: 83200 - 83399 -->
```

Comment:
Why define a range of 200 IDs for a file with three rules? We could change this to
```
<!-- ID: 83200 - 83202 -->
```
or 
```
<!-- ID: 83200 - 83299 -->
```